checker.cpp

```diff
@@ -3,44 +3,63 @@
 #include <unistd.h>
 using namespace std;
 
-int vitalsOk(float temperature, float pulseRate, float spo2) {
-  if(temperature > 102 || temperature < 95) {
-    cout << "Temperature critical!\n";
-    for (int i = 0; i < 6; i++)
+int callSleep()
+{
+     for (int i = 0; i < 6; i++)
     {
       cout << "\r* " << flush;
       sleep(1);
       cout << "\r *" << flush;
       sleep(1);
     }
-
     return 0;
-  } else if(pulseRate < 60 || pulseRate > 100) {
-    cout << "Pulse Rate is out of range!\n";
-    for (int i = 0; i < 6; i++)
+}
+
+int isTemperatureOk(float temperature)
+{
+    if(temperature > 102 || temperature <95)
```

Comment:
Try isolating the comparison into a different function, to accommodate addition of warnings in one place